src/main/java/by/company/servicesqualitymonitoring/category/CategoryMapper.java

```diff
@@ -0,0 +1,13 @@
+package by.company.servicesqualitymonitoring.category;
+
+import org.mapstruct.Mapper;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface CategoryMapper {
+
+    Category convert(CategoryDto categoryDto);
```

Comment:
sometimes I saw following naming:
    convertBoToDto
    convertDtoToBo
or you can create generic interface.